internal/builder/builder.go

```diff
@@ -417,6 +417,37 @@ func addBuildpacks(logger logging.Logger, tmpDir string, image imgutil.Image, ad
 			if existingBPInfo.LayerDiffID == diffID.String() {
 				logger.Debugf("Buildpack %s already exists on builder with same contents, skipping...", style.Symbol(bpInfo.FullName()))
 				continue
+			} else {
+
+				bpWhiteoutsTmpDir := filepath.Join(tmpDir, strconv.Itoa(i)+"_whiteouts")
+				if err := os.MkdirAll(bpWhiteoutsTmpDir, os.ModePerm); err != nil {
+					return errors.Wrap(err, "creating buildpack whiteouts temp dir")
+				}
+
+				deletedMaps := map[string][]byte{
+					filepath.Join(
+						cnbDir,
+						"buildpacks",
```

Comment:
Can you make that a file `var`, similar to `cnbDir`?

Reply:
I will deal with them ASAP.